mayastor/src/core/env.rs

```diff
@@ -199,6 +213,9 @@ impl Default for MayastorEnvironment {
     fn default() -> Self {
         Self {
             config: None,
+            enable_grpc: false,
+            grpc_addr: "None".into(),
+            grpc_port: "None".into(),
```

Comment:
Why don't we pick a new port number here? Just like mayastor-agent has a default port used for grpc, the mayastor should have a default port number different from mayastor-agent, to make it simple to run both concurrently until mayastor grpc server is removed from mayastor-agent.

Reply:
Yep, that's exactly what happens.  The "None" string here gets overwritten by the defaults from the structopt, which as can be seen elsewhere use a new port, 10125.  The invalid value is here as a sort of guard - if the overwriting is missed out, something will choke further down.